openwisp_monitoring/check/base/models.py

```diff
@@ -91,3 +94,22 @@ def auto_ping_receiver(sender, instance, created, **kwargs):
                 object_id=str(instance.pk),
             )
         )
+
+
+def auto_config_modified_receiver(sender, instance, created, **kwargs):
```

Comment:
instead of referring to `config_modified`, let's refer to `config_check`, follow the same suggestion for all other occurrences